### PR TITLE
(DEV-121) Upgrade to RN 58.5, TAP core 0.8.1, and updated prayer resolver

### PR DESCRIPTION
**Pull request by IsaacHardy**
### What does this PR do, or why is it needed?
Updates us to RN 58.5, TAP core 0.8.1, and upgrades the prayer resolver for grabbing a person
### What design trade-offs/decisions were made?
n.a.
### How do I test this PR?
Query to test:
```
# Write your query or mutation here
query {
  getPublicPrayerRequests {
    id
    firstName
    lastName
    text
    createdByPersonAliasId
    campusId
    categoryId
    flagCount
    prayerCount
    person {
      id
      firstName
      lastName
    }
  }
}
```
### What automated tests did you write?
plenty...

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
@IsaacHardy 
Screen Shot 2019-03-27 at 2 04 48 PM

**Comment by redreceipt** [comment]
... also every time I `yarn start` it rewrites fragment types: 

```
~/Documents/Projects/ApollosProject (prayer-resolver *) $ git diff
diff --git a/packages/apolloschurchapp/src/client/fragmentTypes.json b/packages/apolloschurchapp/src/client/fragmentTypes.json
index 6d09200..c2119cc 100644
--- a/packages/apolloschurchapp/src/client/fragmentTypes.json
+++ b/packages/apolloschurchapp/src/client/fragmentTypes.json
@@ -1 +1 @@
-{"__schema":{"types":[{"kind":"INTERFACE","name":"Node","possibleTypes":[{"name":"ContentChannel"},{"name":"Person"},{"name":"PrayerRequest"},{"name":"Pass"},{"name":"Campus"},{"name":"UniversalContentItem"},{"name":"DevotionalContentItem"},{"name":"MediaContentItem"},{"name":"ContentSeriesContentItem"}]},{"kind":"INTERFACE","name":"ContentItem","possibleTypes":[{"name":"UniversalContentItem"},{"name":"DevotionalContentItem"},{"name":"MediaContentItem"},{"name":"ContentSeriesContentItem"}]},{"kind":"INTERFACE","name":"Media","possibleTypes":[{"name":"ImageMedia"},{"name":"VideoMedia"},{"name":"AudioMedia"}]},{"kind":"INTERFACE","name":"MediaSource","possibleTypes":[{"name":"ImageMediaSource"},{"name":"VideoMediaSource"},{"name":"AudioMediaSource"}]},{"kind":"INTERFACE","name":"Sharable","possibleTypes":[{"name":"SharableContentItem"}]}]}}
\ No newline at end of file
+{"__schema":{"types":[{"kind":"INTERFACE","name":"Node","possibleTypes":[{"name":"ContentChannel"},{"name":"Person"},{"name":"Campus"},{"name":"PrayerRequest"},{"name":"Pass"},{"name":"UniversalContentItem"},{"name":"DevotionalContentItem"},{"name":"MediaContentItem"},{"name":"ContentSeriesContentItem"}]},{"kind":"INTERFACE","name":"ContentItem","possibleTypes":[{"name":"UniversalContentItem"},{"name":"DevotionalContentItem"},{"name":"MediaContentItem"},{"name":"ContentSeriesContentItem"}]},{"kind":"INTERFACE","name":"Media","possibleTypes":[{"name":"ImageMedia"},{"name":"VideoMedia"},{"name":"AudioMedia"}]},{"kind":"INTERFACE","name":"MediaSource","possibleTypes":[{"name":"ImageMediaSource"},{"name":"VideoMediaSource"},{"name":"AudioMediaSource"}]},{"kind":"INTERFACE","name":"Sharable","possibleTypes":[{"name":"SharableContentItem"}]}]}}
```

**Comment by IsaacHardy** [comment]
@redreceipt Ready again.

**Comment by redreceipt** [comment]
@IsaacHardy still writing over `fragmentTypes.json` every time a I `yarn start`, something isn't quite right

**Review by redreceipt — APPROVED** [review]
🤘

**Review by redreceipt — APPROVED** [review]
we're good with frags now, I didn't have your latest changes 🤘 🤘 🤘

**Review by redreceipt — APPROVED** [review]
still looks good 🤘

**Review by cdanwards — APPROVED** [review]
